storybook/stories/Stories/Code/WorkspaceItem.elm

```diff
@@ -0,0 +1,229 @@
+module Stories.Code.WorkspaceItem exposing (..)
+
+import Browser
+import Code.Syntax exposing (..)
+import Code.Workspace.WorkspaceItem exposing (Item, Msg(..), WorkspaceItem(..), decodeList, fromItem)
+import Code.Workspace.Zoom exposing (Zoom(..))
+import Dict
+import Helpers.Layout exposing (col)
+import Helpers.ReferenceHelper exposing (sampleReference)
+import Html exposing (Html)
+import Json.Decode exposing (decodeString)
+import UI.ViewMode
+
+
+type alias Model =
+    { zoom : Zoom
+    }
+
+
+main : Program () Model Msg
+main =
+    Browser.element
+        { init = \_ -> ( { zoom = Near }, Cmd.none )
+        , view = view
+        , update = update
+        , subscriptions = \_ -> Sub.none
+        }
+
+
+update : Msg -> Model -> ( Model, Cmd Msg )
+update msg model =
+    case msg of
+        UpdateZoom _ zoom ->
+            ( { model | zoom = zoom }, Cmd.none )
+
+        _ ->
+            ( model, Cmd.none )
+
+
+view : Model -> Html Msg
+view model =
+    let
+        decodeResult : Result Json.Decode.Error (List Item)
+        decodeResult =
+            let
+                result =
+                    decodeString (decodeList sampleReference) incrementGetDefinitionResponse
```

Comment:
@hojberg 
Do you think this way, which is to decode hard-coded JSON strings, is ok for now?
I couldn't find a way to make entities manually, without changing the access controls (`exposing`s) in ui-core. Json string decoding was the only way I could find to make entities.

When we want to see other codes, we can replace the JSON string with others, which we can get by accessing UCM API.

Reply:
Yeah it seems ok for now